sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs

```diff
@@ -221,8 +220,7 @@ public virtual async Task<Response<NamespaceProperties>> GetNamespacePropertiesA
                     null,
                     false,
                     cancellationToken).ConfigureAwait(false);
-                var result = await ReadAsString(response).ConfigureAwait(false);
-                NamespaceProperties properties = NamespacePropertiesExtensions.ParseFromContent(result);
+                NamespaceProperties properties = await NamespacePropertiesExtensions.ParseResponseAsync(response, _clientDiagnostics).ConfigureAwait(false);
```

Comment:
The class name is a bit odd.   Definitely not worth changing, but I wonder if this would have been more intuitive as a set of extensions for the response, since it's not really extending the namespace properties and similar types.